codec-http/src/main/java/io/netty5/handler/codec/http/websocketx/extensions/WebSocketServerExtensionHandler.java

```diff
@@ -132,9 +132,7 @@ public Future<Void> write(final ChannelHandlerContext ctx, Object msg) {
                             }
                         };
 
-                        if (newHeaderValue != null) {
-                            headers.set(HttpHeaderNames.SEC_WEBSOCKET_EXTENSIONS, newHeaderValue);
-                        }
+                        headers.set(HttpHeaderNames.SEC_WEBSOCKET_EXTENSIONS, newHeaderValue);
```

Comment:
This is a behaviour change... just wanted to highlight this.

Reply:
The `newHeaderValue` comes from `WebSocketExtensionUtil.computeMergeExtensionsHeaderValue` which never returns `null`.